bluez-async/src/lib.rs

```diff
@@ -631,6 +634,19 @@ impl BluetoothSession {
         )
     }
 
+    fn device_with_timeout(&self, id: &DeviceId, timeout: Duration) -> impl OrgBluezDevice1 + Introspectable + Properties {
+        let mut local_timeout_value =  DBUS_METHOD_CALL_TIMEOUT; // use default value
+        if timeout.as_secs() <= DBUS_METHOD_CALL_MAX_TIMEOUT.as_secs() {
+            local_timeout_value = timeout; // take passed external timeout value
+        }
```

Comment:
Can't you just do

```rust
let timeout = min(timeout, DBUS_METHOD_CALL_MAX_TIMEOUT);
```

? Capping the timeout at the max seems better than falling back to the default value.

Reply:
fixed to
`let local_timeout_value = timeout.min(DBUS_METHOD_CALL_MAX_TIMEOUT);`